### DISCUSSION TITLE
Output esperado parte 2

### DISCUSSION THREAD

**Discussion by Catalinapadilla** [IIC2133-PUC/2024-1#140] [Tarea 2] [answered]
Hola!
Quería saber si para la parte 2 se tomarán todas las posibles combinaciones de paquetes como correctas, porque, si revisé bien, hay tests que permiten más de una solución, como por ejemplo medium-1

**Comment by sebacarrascop** [comment, marked as answer]
Hola!, tienes razón, actualmente hay tests que permiten más de una solución por un pequeño error en el generador!, gracias por avisar. Ya arreglamos el error y pronto vamos a generar nuevos test que solo tengan una solución (que es como debería ser por lo indicado en el enunciado), por acá mismo te aviso cuando subamos los test corregidos

**Reply by sebacarrascop** [reply]
Están arriba los nuevos!
